RecurringBudgetBundle -> Update for Kimai 2 (#494)
kimai/www.kimai.org: _includes/store/pcsg-recurring-budget-bundle.md

> A Kimai plugin that automatically adds a recurring money or time budget to your projects with a custom interval. Set

Reviewer: ```suggestion
Automatically add a recurring money or time budget to your projects with a custom interval. Set
```

up your projects to receive a fixed amount of budget on a regular basis (e.g. 05:00 hours every month). Ideal for
contracts with a regular budget bonus (e.g. maintenance agreements, SLAs etc.).

## Features
* Automatic budget credits on a per-project basis (time or money)
* Custom budget interval (every month, every 90 days or whatever you desire)
* Detailed budget history view for all projects and budget intervals in the Kimai backend
* Current budget view for all projects in the Kimai backend
* Export budget summaries for specific time periods with all budget details for your documents or your customers

> * **NEW for Kimai 2**
>   * Automatic budget expiration: You can now configure your projects to expire unused budget after a set amount of intervals
>   * Recalculate budget intervals via frontend: The budgets history view now offers the option to recalculate a budget interval directly

Reviewer: ```suggestion
* Automatic budget expiration: You can now configure your projects to expire unused budget after a set amount of intervals
* Recalculate budget intervals via frontend: The budgets history view now offers the option to recalculate a budget interval directly
```

## Requirements

> * Kimai, **v2.20.0** or higher

Reviewer: ```suggestion
* Kimai 2.20.0 or higher
```

A Kimai plugin that automatically adds a recurring money or time budget to your projects with a custom interval. Set
up your projects to receive a fixed amount of budget on a regular basis (e.g. 05:00 hours every month). Ideal for
contracts with a regular budget bonus (e.g. maintenance agreements, SLAs etc.).

> # Now ready for Kimai 2!

Reviewer: Mentions of versions always confuse users. I have a sad history of that. I will remove it.
```suggestion
```
